src/components/application_manager/include/application_manager/application.h

```diff
@@ -786,6 +791,23 @@ class Application : public virtual InitialApplicationData,
    */
   virtual uint32_t GetAvailableDiskSpace() = 0;
 
+  /**
+   * @breif Allows to save mobile's command smart object in order to perform
+   * this command later.
+   * @param mobile_message the message smart_object.
+   */
+  virtual void PushMobileMessage(
```

Comment:
@LitvinenkoIra could you please comment why do we need to save/resume some mobile messages for applications? As I can see from requirements it is enough just to send notification with last known driver distraction state